src/App.module.css

```diff
@@ -2,11 +2,94 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
 }
 
 .header {
   font-family: Arial, Helvetica, sans-serif;
   font-size: 2rem;
   font-weight: bold;
+  justify-content: center;
+}
+
+body {
+  font-family: "Georgia", serif;
+  background-color: #f9f9f9;
+  margin: 0;
+  padding: 20px;
+  line-height: 1.6;
+  color: #555;
+}
+
+h1 {
+  font-size: 2em;
+  color: #333;
+  margin-bottom: 20px;
+}
+
+button {
+  background-color: #ff7043;
+  color: white;
+  padding: 10px 20px;
+  margin-bottom: 20px;
+  border: none;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 1em;
+  transition: background-color 0.3s;
+}
+
+button:hover {
+  background-color: #ff5722;
+}
+
+div {
+  max-width: 1000px;
+  margin: auto;
+  background: white;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+  margin-top: 20px;
+}
+
+th,
+td {
+  padding: 12px 15px;
+  text-align: left;
+  border-bottom: 1px solid #eee;
+}
+
+th {
+  background-color: #ffebee;
+  color: #333;
+}
+
+tr:hover {
+  background-color: #ffecb3;
+}
+
+h1 {
+  text-align: center;
+}
+
+button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+.FlightTable .flight-time-column {
```

Comment:
CSS convention is to name classes with lowercase and dashes between words. [Here's a cool article on that.](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)